mlflow/utils/uri.py

```diff
@@ -64,3 +65,67 @@ def get_uri_scheme(uri_or_path):
         return extract_db_type_from_uri(uri_or_path)
     else:
         return scheme
+
+
+def append_to_uri_path(uri, *paths):
+    """
+    Appends the specified POSIX `paths` to the path component of the specified `uri`.
+
+    :param uri: The input URI, represented as a string.
+    :param paths: The POSIX paths to append to the specified `uri`'s path component.
+    :return: A new URI with a path component consisting of the specified `paths` appended to
+             the path component of the specified `uri`.
+
+    >>> uri1 = "s3://root/base/path?param=value"
```

Comment:
`uri = ` ?

Reply:
Good catch! Will file a followup PR to fix this, since we're close to branch cut.